bigquery/cloud-client/src/main/java/com/example/bigquery/AuthSnippets.java

```diff
@@ -0,0 +1,90 @@
+/*
+  Copyright 2017, Google, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.example.bigquery;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Dataset;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Examples for authenticating to Google BigQuery.
+ *
+ * <p>See: https://cloud.google.com/bigquery/authentication
+ */
+public class AuthSnippets {
+
+  // [START explicit_service_account]
+  public static void explicit() throws IOException {
+    // Load credentials from JSON key file.
+    GoogleCredentials credentials;
+    File credentialsPath = new File("service_account.json");  // TODO: update to your key path.
```

Comment:
I'd be happy if he just mentioned GOOGLE_APPLICATION_CREDENTIALS, I suspect the page mentions it.